pkg/kv/mem/store.go

```diff
@@ -43,17 +43,17 @@ func (d *Driver) Open(_ context.Context, _ kvparams.KV) (kv.Store, error) {
 }
 
 func combinedKey(partitionKey, key []byte) string {
-	return fmt.Sprintf("%s_%s", partitionKey, key)
+	return kv.FormatPath(string(partitionKey), string(key))
 }
 
 func keyFromCombinedKey(combinedKey string) []byte {
 	//nolint:gomnd
-	return []byte(strings.SplitN(combinedKey, "_", 2)[1])
+	return []byte(strings.SplitN(combinedKey, kv.PathDelimiter, 2)[1])
```

Comment:
Isn't `PathDelimiter` a simple `/`? What if it's in the partitionKey as well?

Reply:
I think we can defer this conversation since we're going to re-implement mem soon. This is just so the tests will pass